tests/console/gd.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: gd regression test
+#   * Convert gif image to a gd2 image
+#   * Convert gif image to a compressed gd2 image
+#   * Check if compressed image is smaller than original
+#   * Convert gd2 image to various other formats
+#   * Check the output of webpng
+# Maintainer: Felix Niederwanger <felix.niederwanger@suse.de>
+
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    select_console 'root-console';
```

Comment:
I'd use serial console as it would be faster, but it's up to you :)

Reply:
Good idea! I'll change it to serial console

Reply:
I think I leave it for now but keep it in mind for the future. Thanks for letting me know